proposals/NNNN-android-manager-listener-update.md

```diff
@@ -175,15 +175,15 @@ public class Updater {
 
 ## Potential downsides
 
-From a technical point of view there is no downside to the added features. However as Java interfaces force you to implement every single method which can be distracting. 
+From a technical point of view there is no downside to the proposed interface methods. However, as Java interfaces force you to implement every single method, it can be distracting. 
 
 ## Impact on existing code
 
-The additional methods in the interface have impact to implementations of app developers and may be distracting. Besides of that there's no existing code changed.
+The additional methods in the interface have impact to implementations that app developers may find distracting. Besides of that there's no existing code changes.
```

Comment:
Update to:
Besides that there's no existing code changes.

Reply:
@jordynmackool fixed